crates/rpc/rpc/src/eth/error.rs

```diff
@@ -18,31 +18,11 @@ pub(crate) enum EthApiError {
     #[error("Invalid transaction signature")]
     InvalidTransactionSignature,
     #[error(transparent)]
-    PoolError(GethCompatPoolError),
+    PoolError(PoolError),
```

Comment:
this needs to be `GethTxPoolError` now, because otherwise we're not using `GethTxPoolError`, right?